src/app/static/src/app/store/filteredData/filteredData.ts

```diff
@@ -58,9 +67,41 @@ export const initialSelectedFilters: SelectedFilters = {
     }
 };
 
+export const initialSelectedChoroplethFilters: SelectedChoroplethFilters = {
+    sex: null,
+    age: null,
+    survey: null,
+    updateByType: function(type: FilterType, value: FilterOption) {
```

Comment:
Shouldn't this be a mutation? Pretty sure you shouldn't be mutating state outside of mutations.